bulbs/special_coverage/filters.py

```diff
@@ -0,0 +1,20 @@
+from bulbs.utils.filters import CaseInsensitiveBooleanFilter
+
+from bulbs.utils.methods import get_query_params, today
+
+
+class SpecialCoverageFilter(CaseInsensitiveBooleanFilter):
+    """
+    Includes the default boolean_filter, but replaces the active and inactive states to represent
+    whether the current_date is within a SpecialCoverage's start_date & end_date range.
+    """
+    def filter_queryset(self, request, queryset, view):
+        queryset = super(SpecialCoverageFilter, self).filter_queryset(request, queryset, view)
+        query_params = get_query_params(request)
+        if "active" in query_params:
+            value = query_params.get("active").lower()
+            if value == "true":
+                queryset = queryset.filter(start_date__lte=today(), end_date__gte=today())
```

Comment:
I know this is being picky, but should save today() result to local variable and use for both params, to avoid (extremely) slim chance of today() returning two different values